app/views/locations/show.html.erb

```diff
@@ -1,8 +1,11 @@
-<p id="notice"><%= notice %></p>
+<% content_for :sidebar do %>
+  <%= sidebar_button_link 'Edit', edit_location_path(@location) %>
+  <%= sidebar_button_link 'Back', locations_path %>
+<% end %>
 
 <p>
   <b>Department:</b>
-  <%= @location.department.name %>
+  <%= @location.department&.name %>
```

Comment:
Defensive coding (Ruby 2.3's new Safe Navigation Operator).  The view blew up when I happened to test it on a location whose department wasn't set properly.  Happy to remove if we're going to ensure that the value is set. I think to_s would also work, but somehow the view needs to be protected.